cylc/flow/task_events_mgr.py

```diff
@@ -803,13 +803,13 @@ def _process_message_check(
                     f"[{itask}] "
                     f"{self.FLAG_RECEIVED_IGNORED}{message}{timestamp}"
                 )
-
-            else:
+                return False
```

Comment:
@MetRonnie  This isn't the same as the change on master - It's the response to this comment: https://github.com/cylc/cylc-flow/pull/5927#discussion_r1453767051. I wanted to see what CI made of it. [This is the commit where I changed it](https://github.com/cylc/cylc-flow/pull/5927/commits/d4514f4a37c596732b402419ca3f276772c6d387) from what's on master.